invenio_logging/structlog.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import sys
+from logging import StreamHandler
+
+import structlog
+
+
+class LoggerFactory:
+    @staticmethod
+    def configure_logging(
```

Comment:
comment/question: maybe another way to integrate would be to follow the `structlog` guides for specific frameworks like e.g. [Flask](https://www.structlog.org/en/stable/frameworks.html#flask) and [Celery](https://www.structlog.org/en/stable/frameworks.html#celery). Especially for Celery that will really allow us to track individual tasks/jobs (though we'll need some central attribute/key to be able to group things logically).